lib/features/steps/runner_steps.rb

```diff
@@ -8,6 +8,21 @@
   Runner.environment[key] = value
 end
 
+# Verifies that an environment variable is set to a given value.
+#
+# @step_input key [String] The environment variable
+# @step_input value [String] The intended value of the environment variable
+Then('the environment variable {string} equals {string}') do |key, expected_value|
+  assert_equal(expected_value, Runner.environment[key])
+end
```

Comment:
I don't think this should be checking `Runner.environment` as it doesn't necessarily represent the actual environment variables

```suggestion
Then('the environment variable {string} equals {string}') do |key, expected_value|
  assert_equal(expected_value, ENV[key])
end
```